src/type/__tests__/enumType.js

```diff
@@ -59,7 +66,31 @@ describe('Type System: Enum Values', () => {
     }
   });
 
-  var schema = new GraphQLSchema({ query: QueryType });
+  var MutationType = new GraphQLObjectType({
+    name: 'Mutation',
+    fields: {
+      favoriteEnum: {
+        type: FavoriteType,
+        args: { color: { type: ColorType } },
+        resolve(value, color) { return color; }
+      }
+    }
+  });
+
+  var schema = new GraphQLSchema({ query: QueryType, mutation: MutationType });
+
+  it('accepts enum literals as input arguments to mutations',
```

Comment:
Could you move this below to the equivalent test ensuring variables are coerced correctly for query operations on line 135?